app/views/posts/index.html.erb

```diff
@@ -1 +1,34 @@
-<h1>All Posts</h1>
\ No newline at end of file
+<main class='single-post-container'>
+  <section class='user-header'>
+    <div class='user-photo'>
+      <img src= <% @user.photo %> class='user-image'>
+    </div>
+    <div class='user-info'>
+      <h2><%= @user.name %></h2>
+        <p>Number of posts: <%= @user.posts_counter %></p>
+    </div>
+  </section>
+
+  <section>
+    <% if @user.last_posts.blank? %>
+      <h2>There are no posts. Please add a post.</h2>
+    <% else %>
+      <% @user.posts.each do |post| %>
+      <div class='single-post'>
+        <h3>Post: # <%= link_to post.id, user_post_path(@user, post) %></h3>
+        <p><%= post.text %></p>
+        <p class='counter'>Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+
+        <% post.last_comments.each do |comment| %>
+          <div class='comments'>
+            <p><%= comment.author.name %>:
+            <p><%= comment.text %>
+          </div>
+        <% end %>
+      </div>
+      <% end %>
+    <% end %>
+  </section>
+
+  <a class='pagination-btn' href='#'>Pagination</a>
+</main>
```

Comment:
- [ ] it would be better to add spacing margin and styles to put the elements as in the wireframe 
- the profile picture is not displayed and it should I the same line as the users' info in order to match the wireframe 
- the comments and likes counter should be on the left of the card
- and the pagination button should be centered
- so that you match the wireframe design 
- Yours 

<img width="307" alt="Screenshot 2023-06-02 at 11 40 33" src="https://github.com/lugard1/blogapp/assets/61790587/0943d06b-0e93-44d4-8562-01ce40e5d2bc">


- Design 
<img width="449" alt="Screenshot 2023-06-02 at 11 27 03" src="https://github.com/lugard1/blogapp/assets/61790587/8247ef01-fc11-4e26-95e5-a6b92abbe69a">